changes/861.changed

```diff
@@ -0,0 +1 @@
+- [chore] `towncrier` is now used for changelog management ([#861](https://github.com/Substra/substra-backend/pull/861))
```

Comment:
Do we need to specify "chore" in the message ?

Reply:
(asking for my future self)

Reply:
no, the only thing that matters is the name of the file (has to end with `.added`, `.fixed`, `.removed` or `.changed`

you can put whatever you want inside the file, including nested multiline lists, code snippets...

Reply:
okay I see thx